### PR TITLE
BaSD-MS-35: Buttons

### PR DESCRIPTION
**Pull request by ignacioaristo**
Shared buttons were created and added to every entity.

- Create button.
- Delete button.

image

### CODE REVIEW

**Review by leimeterjoaquin — APPROVED** [review]
It looks great

File: src/Components/Admins/index.js [hunk 1]
```diff
@@ -72,7 +74,10 @@ const Admins = () => {
         text="Are you sure you want to delete the admin selected?"
       ></Modal>
       <ErrorMessage show={showErrorMessage} close={closeError} text={errorMessageText} />
-      <h2 className={styles.header}>Admins</h2>
+      <div className={styles.titleAndButton}>
+        <h3>Admin</h3>
+        <Button onClick={() => addAdmin()} value="Admin" />
```
Comment (aquilesmariani) [inline src/Components/Admins/index.js:79]: If you don't need to send any extra parameter through this place you can do this:
```suggestion
        <Button onClick={addAdmin} value="Admin" />
```

File: src/Components/Councelors/index.js [hunk 2]
```diff
@@ -72,7 +74,10 @@ const Councelor = () => {
         text="Are you sure you want to delete the counselor selected?"
       ></Modal>
       <ErrorMessage show={showErrorMessage} close={closeError} text={errorMessageText} />
-      <h2 className={styles.header}>Counselors</h2>
+      <div className={styles.titleAndButton}>
+        <h3>Councelors</h3>
```
Comment (aquilesmariani) [inline src/Components/Councelors/index.js:78]: We should rename everything so we don't have this typo. But let's put it as tech debt
```suggestion
        <h3>Counselors</h3>
```

**Review by aquilesmariani — APPROVED** [review]
Looks great! I've left two comments so you can change and merge (if you realize it's working properly)

**Pull request merged**